recipes/docker-py/meta.yaml

```diff
@@ -27,7 +27,7 @@ requirements:
     - six >=1.4.0
     - websocket-client >=0.32.0
     - backports.ssl_match_hostname >=3.5  # [not py35]
-    - ipaddress ==1.0.16  # [not py33 and not py34 and not py35]
+    - ipaddress ==1.0.16  # [py2k]
```

Comment:
So, actually I would be tempted to do `# [py<33]` as that is more inline with the author's intent.